test/freenet/support/MultiValueTableTest.java

```diff
@@ -177,11 +176,11 @@ public void testContainsElement() {
 			createSampleKeyMultiVal(sampleKeyNumber,sampleMaxValueNumber,sampleIsRandom);
 		methodMVTable = fillMultiValueTable(sampleObjects);
 		Iterator<Object> iter;
-		for(int i=0;i<sampleObjects.length;i++) {
-			iter = ((List<Object>)(sampleObjects[i][1])).iterator();
-			assertFalse(methodMVTable.containsElement(sampleObjects[i][0],new Object()));
-			while(iter.hasNext())
-				assertTrue(methodMVTable.containsElement(sampleObjects[i][0],iter.next()));
+		for (Object[] sampleObject : sampleObjects) {
```

Comment:
Please cast `sampleObjects` to a `List<Object>[][]`; this way we can have a more specific type for the loop variable and get rid of the cast in the line below.